CHANGELOG.md

```diff
@@ -43,6 +43,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1197](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1197))
 - Add metric instumentation for flask
   ([#1186](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1186))
+- `opentelemetry-util-http` Add support for sanitizing HTTP header values.
```

Comment:
Please move this to unreleased section

Reply:
Yeah, this keeps ending up in the wrong place after `main` gets merged in to my branch.

Reply:
Fixed.